logging/log_collection_forwarding/log-forwarding.adoc

```diff
@@ -180,7 +178,7 @@ include::modules/loki-rate-limit-errors.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../logging/cluster-logging-exported-fields.adoc#cluster-logging-exported-fields-kubernetes_cluster-logging-exported-fields[Log Record Fields].
+* xref:../../logging/cluster-logging-exported-fields.adoc#cluster-logging-exported-fields-kubernetes_cluster-logging-exported-fields[Log Record Fields].
```

Comment:
This extraneous period is existing but might as well remove it while making changes?
```suggestion
* xref:../../logging/cluster-logging-exported-fields.adoc#cluster-logging-exported-fields-kubernetes_cluster-logging-exported-fields[Log Record Fields]
```